docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc

```diff
@@ -0,0 +1,301 @@
+This section will discuss about the configuration of your AeroHIVE AP in order to use it with our configured PacketFence environment.
+
+Connect APs and Controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+First thing to do is to connect the HiveManager. Like you saw in Step 2, make sure to connect the AP on the TRUNK port configured for it.
+
+Provision the HiveManager
+^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the HiveManager might be empty, add your AP on it via it's serial number. If necessra update the firmware of the HiveManager and the AP.
+
+NOTE: Please refer to the AeroHIVE Quick Devices Start Guide to add devices: http://docs.aerohive.com/330000/docs/help/english/documentation/AerohiveDevicesQuickStart.pdf
+
+NOTE: Please refer to the following guide to learn more about the HiveManager: http://docs.aerohive.com/330000/docs/help/english/documentation/Aerohive_HiveManagerNG_GettingStartedGuide.pdf
+
+Setup the SSIDs
+^^^^^^^^^^^^^^^
+For this guide, you will show you how to configure one open SSID with RADIUS mac filtering, and one secure SSID using WPA2-Enterprise.
+
+Login to your HiveManager using the admin credentials. You should see the Dashboard. Click on the Configure pane, and follow the following steps.
+
+Configure the AAA Servers
++++++++++++++++++++++++++
+On the left menu, click on Advanced Configuration, then Authentication, then AAA Client Settings. We need to add the PacketFence RADIUS server here. Click on the New button, and provide follow this configuration.
+
+image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+
+Click OK to save the server. You should see the RADIUS server in the list.
+
+Configure the Open SSID
++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those information.
+
+image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
+
+Click OK to save the SSID.
+
+Configure the Secure SSID
++++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those inforamtion.
+
+image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
+
+Click OK to save the SSID.
+
+Configure User Profile
+^^^^^^^^^^^^^^^^^^^^^^
+You will now need to create user profile matching your VLAN assignements, the "Default VLAN" have been created before to match the desired VLAN number.
+
+Create user profile as follow:
+
+image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+
+image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
+
+Configure SNMP
+^^^^^^^^^^^^^^
+You now need to configure the SNMP to add a valid read community string. Click on the Configure pane, and on Advanced Configuration, then Management Policies, then SNMP Assignements. Now click on New and follow this configuration.
```

Comment:
_Click on the 'Configure' pane, and on 'Advanced Configuration', then 'Management Policies', then 'SNMP Assignements'. Now click on 'New'_